src/benchcab/utils/meorg.py

```diff
@@ -0,0 +1,96 @@
+"""Utility methods for interacting with the ME.org client."""
+from benchcab.internal import MEORG_CLIENT
+from meorg_client.client import Client as MeorgClient
+from hpcpy import get_client
+import benchcab.utils as bu
+import os
+from glob import glob
+
+def do_meorg(config: dict, upload_dir: str, benchcab_bin: str, benchcab_job_id: str):
+    """Perform the upload of model outputs to modelevaluation.org
+
+    Parameters
+    ----------
+    config : dict
+        The master config dictionary
+    upload_dir : str
+        Absolute path to the data dir for upload
+    benchcab_bin : str
+        Path to the benchcab bin, from which to infer the client bin
+
+    Returns
+    -------
+    bool
+        True if successful, False otherwise
+    """
+
+    logger = bu.get_logger()
+
+    model_output_id = config.get("fluxsite").get("meorg_model_output_id", False)
```

Comment:
Please replace `get` with regular dictionary indexing, default values in the config should be initialised [here](https://github.com/CABLE-LSM/benchcab/blob/main/src/benchcab/config.py#L74)

Reply:
Done, although I think it would be good practice to use the `.get` approach in development with a default for the null case. Perhaps we could override the get method to do a lookup in place of a missing key rather than manually unpacking everything as it currently is in `internal.py`

I think we should flag this reliance on `internal.py` to set up defaults for the portability project. It seems to me that we could essentially configure all defaults with the schema itself...